src/style.css

```diff
@@ -1 +1,74 @@
 /* Styles go here */
+* {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  align-items: center;
+  width: 1200px;
+  height: 60px;
+  background-color: azure;
+}
+
+.logo {
+  margin-top: 3px;
+
+}
+
+/* .menu {
+  margin-left: 275px;
+} */
+
+.nav__items {
+  display: inline-block;
+  margin-left: 5px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  transition: color 0.3s;
+  font-weight: bold;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  padding: 0 5px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  /* color: #00acdc;
+  padding-bottom: 20px;
+  border-bottom: 4px solid #00acdc;
+  border-bottom-left-radius: 4px;
+  border-bottom-right-radius: 4px; */
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 40px;
```

Comment:
Better to use left and right coordinates to stretch the element. In this case it will work with any width.